superset/commands/database/create.py

```diff
@@ -148,26 +127,6 @@ def run(self) -> Model:
 
         return database
 
-    def add_schema_permissions(
```

Comment:
Searched for `add_schema_permissions` in the code and couldn't find any other occurrence. Happy to leave it there if we feel like it could be useful in the future tho!